### PR TITLE
Bank account management in new window; fixed code review errors;

### PR DESCRIPTION
**Pull request by AndYavorskiy**
Set/Edit organization logo;
Set org logo on register organization page;
Fixed tests;
Created common-message.storage.ts to store all messages user can get;
Created default-configuration.storage.ts to store default configurations for all project.

### CODE REVIEW

File: FundTrack.BLL/Concrete/OfferedItemService.cs [hunk 1]
```diff
@@ -225,37 +201,40 @@ public OfferedItemViewModel InitializeOfferedItemViewModel(OfferedItem item)
...
+            Dictionary<OfferedItemImage, Task<string>> imageTastDictionary = new Dictionary<OfferedItemImage, Task<string>>();
+
+            foreach (var item in imagesInBase64code)
             {
-                var newImage = new OfferedItemImage
+                var newImage = new OfferedItemImage()
                 {
-                    IsMain = thing.IsMain,
-                    OfferedItemId = item.Id
+                    IsMain = false,
```
Comment (unclefrost) [inline FundTrack.BLL/Concrete/OfferedItemService.cs]: not needed, booleans are false by default

File: FundTrack.BLL/Concrete/OfferedItemService.cs [hunk 2]
```diff
@@ -225,37 +201,40 @@ public OfferedItemViewModel InitializeOfferedItemViewModel(OfferedItem item)
...
+            foreach (var element in imageTastDictionary)
             {
                 element.Key.ImageUrl = element.Value.Result;
             }
-
-            this._unitOfWork.OfferImagesRepository.CreateMany(d.Keys.ToList());
-            return model.Image;
+            if (imageTastDictionary.Keys.First() != null)
+            {
+                imageTastDictionary.Keys.First().IsMain = true;
```
Comment (unclefrost) [inline FundTrack.BLL/Concrete/OfferedItemService.cs]: not sure if this is a good place to decide what image is "main". Shouldn't API client (end-user) decide which of the images is the main one? Consider assigniong value of this property on the UI.

File: FundTrack.BLL/Concrete/OrganizationRegistrationService.cs [hunk 3]
```diff
@@ -38,6 +42,15 @@ public OrganizationRegistrationViewModel RegisterOrganization (OrganizationRegis
                     if (checkMembership == null)
                     {
                         var organization = new Organization { Name = item.Name, Description = item.Description };
+
+                        if (!String.IsNullOrEmpty(item.Base64Code))
+                        {
+                            var task = _imgManageService.UploadImage(Convert.FromBase64String(item.Base64Code));
```
Comment (unclefrost) [inline FundTrack.BLL/Concrete/OrganizationRegistrationService.cs]: if items do not upload to Azure, this should not be commited. you have added lines of code to the existing business process (registration of organization) that SHOULD upload images during the registation but in fact, they do nothing. If you want to write some piece of code that is not functional instantly,or does not add any business value, but will in future, do not invoke it in the existing busines flow.

File: FundTrack.WebUI/Controllers/OrganizationProfileController.cs [hunk 4]
```diff
@@ -67,5 +67,11 @@ public EditAddressViewModel EditAddress([FromBody] AddressViewModel address)
         {
             return _orgProfileService.EditAddress(address);
         }
+
```
Comment (BogdanLyseiko) [inline FundTrack.WebUI/Controllers/OrganizationProfileController.cs]: rest

**Pull request closed without merging**